Example/SlideController/PageLifeCycleObject.swift

```diff
@@ -37,17 +37,17 @@ extension PageScrollViewLifeCycle_Implementation : PageScrollViewLifeCycle {
         
     }
     
-    func didStartScrolling() {
+    func didStartSliding() {
         
     }
     
-    func didCancelScrolling() {
+    func didCancelSliding() {
         
     }
```

Comment:
We decided to use shorten form `func didCancelSliding() { }` for empty functions implementations or specify it directly in comment in case when it is special intention to make empty function.